test/e2e/targets_api_test.go

```diff
@@ -106,40 +106,3 @@ func TestTargetsAPI_Fanout(t *testing.T) {
 		},
 	})
 }
-
-//nolint:unused
-func targetAndAssert(t *testing.T, ctx context.Context, addr string, state string, want *targetspb.TargetDiscovery) {
```

Comment:
No, please keep it. This function is not used because we disabled the flaky test case for now.

Reply:
See also https://github.com/thanos-io/thanos/pull/4153

Reply:
👍🏽